### ISSUE TITLE
Traning dataset?

### ISSUE THREAD

**Issue by zhangwlgq** [nagadomi/waifu2x#196]
I trained the model using my own dataset of about 2k high-resolution images. But the upscale result is not as good as the pretrained model (photo/scale2.0x_model.json).

Could you please tell me which dataset is used during your training? Or any tricks used in training?

**Comment by nagadomi** [comment]
Did you use our training code?
I published training parameters at https://github.com/nagadomi/waifu2x/blob/master/appendix/train_upconv_7_photo.sh

waifu2x's photo models was trained on the blending dataset of [kou's photo collection](http://photosku.com/photo/category/%E6%92%AE%E5%BD%B1%E8%80%85/kou/) and [ukbench](http://vis.uky.edu/~stewe/ukbench/) (This dataset seems to be gone now).

**Comment by nagadomi** [comment]
Note that  kou's photos are too big and contains a lot of ISO noise. I use these data after resizing.

**Comment by nagadomi** [comment]
Also I published the benchmark results at https://github.com/nagadomi/waifu2x/blob/dev/appendix/benchmark.md
resnet_14l is only available on dev branch.

**Comment by zhangwlgq** [comment]
Yes, I used your training code, I think the only difference is dataset.
Thanks for your information, I will try it soon.

**Comment by zhangwlgq** [comment]
And what is the difference between art and photo model?
I used both of them, it seems the art model works slightly better than the photo model. Though my input images are actually photos.

**Comment by nagadomi** [comment]
training dataset and hyper parameters are different.
